src/basic/os-util.c

```diff
@@ -36,7 +36,7 @@ bool image_name_is_valid(const char *s) {
         return true;
 }
 
-int path_is_extension_tree(const char *path, const char *extension) {
+int path_is_extension_tree(const char *path, const char *extension, bool extension_release_nonstrict) {
```

Comment:
I'd always prefer "positive" ways to name thins instead of "negated" ones. i.e. maybe name this "relax_extension_release_check" or so?